src/TyroSharedContext.tsx

```diff
@@ -282,6 +289,7 @@ const TyroProvider = ({ children, options }: TyroPayContext): JSX.Element => {
     payRequest,
     isPayRequestReady,
     isPayRequestLoading,
+    isWalletPaymentReady,
```

Comment:
do we want to share this to user? If the device does not support the wallet pay this will never be true, and we are already hiding the wallet pay button from paysheet
If we want to share it should we rename it to isWalletPaySupported?